### ISSUE TITLE
batman hangs when using BAT_PAGER

### ISSUE THREAD

**Issue by docwhat** [eth-p/bat-extras#106]
If I `export BAT_PAGER=less` then `batman <page>` hangs on macOS 13 & 14.

I can see (via PS) it running many copies of `bat --language=man...` until it hits a fork limit.

At the moment, I can reproduce at will.

**Comment by tarneaux** [comment]
I tried finding a workaround by setting `$MANPAGER` but had some issues with it. Here's what I ended up with:

```sh
export PAGER="bat -p"
export BAT_PAGER="less -RF --jump-target=.5" # change to your liking

__myman() {
    man $@ | bat -l man -p
}

alias man="__myman"
```

For ZSH completion, put this after the `autoload -U compinit` line in your zshrc:
```zsh
compdef _man __myman
```
Meaning "use the `man` completion for `__myman`".

**Comment by eth-p** [comment]
That was quite the interesting one to debug. Thanks for finding this!

Fixed in commit 205e9c4.

**Comment by eth-p** [comment]
Released in [v2024.02.12](https://github.com/eth-p/bat-extras/releases/tag/v2024.02.12).
